core/src/cubos/core/io/glfw_window.cpp

```diff
@@ -270,9 +272,9 @@ const char* GLFWWindow::getClipboard() const
 
 #ifdef WITH_GLFW
 
-static void keyCallback(GLFWwindow* window, int key, int scancode, int action, int mods)
+static void keyCallback(GLFWwindow* window, int key, int, int action, int mods)
 {
-    GLFWWindow* handler = (GLFWWindow*)glfwGetWindowUserPointer(window);
+    auto* handler = (GLFWWindow*)glfwGetWindowUserPointer(window);
```

Comment:
*ditto* (and for the other 5 cases in this file)
```suggestion
    GLFWWIndow* handler = (GLFWWindow*)glfwGetWindowUserPointer(window);
```